subprojects/launcher/src/main/java/org/gradle/tooling/internal/provider/continuous/ContinuousBuildTriggerHandler.java

```diff
@@ -14,17 +14,17 @@
  * limitations under the License.
  */
 
-package org.gradle.tooling.internal.provider;
+package org.gradle.tooling.internal.provider.continuous;
 
+import org.gradle.deployment.internal.ContinuousExecutionGate;
 import org.gradle.initialization.BuildCancellationToken;
-import org.gradle.initialization.ContinuousExecutionGate;
 import org.gradle.internal.UncheckedException;
 
 import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.TimeUnit;
 
 public class ContinuousBuildTriggerHandler {
-    public static final String QUIET_PERIOD_SYSPROP = "org.gradle.internal.filewatch.quietperiod";
+    public static final String QUIET_PERIOD_SYSPROP = "org.gradle.continuous.quietperiod";
```

Comment:
This is the only "user facing" change. I wonder if we should rename the property at all. WDYT?

Reply:
Doing so would move it to the public API, right? I think given the feedback from our own colleague the other week that would make sense.

Reply:
Well, maybe the name should be `org.gradle.internal.continuous.quietperiod`. When moving it to the public API, I suppose we'd use a `BuildOption` instead of a system property?

Reply:
I am fine with making it public. I'd use a separate PR for that, though.

Reply:
That makes sense.